data/mods/Xedra_Evolved/effects/vampvirus.json

```
@@ -148,7 +149,7 @@
               "condition": { "u_has_trait": "VAMPIRE" },
               "effect": [
                 {
-                  "u_roll_remainder": [ "EYEGLEAM", "STAMINAFORBLOOD", "COMMUNE_NIGHT" ],
+                  "u_roll_remainder": [ "EYEGLEAM", "STAMINAFORBLOOD", "BLOODBANK", "COMMUNE_NIGHT" ],
```

```suggestion
                  "u_roll_remainder": [ "EYEGLEAM", "STAMINAFORBLOOD", "COMMUNE_NIGHT" ],
```
Can this be moved to VAMPIRE3's powers?